### PR TITLE
docs(card): add draggable examples to Card documentation - FE-1611

### PR DESCRIPTION
**Pull request by DipperTheDan**
currently we do not have examples of draggable examples of the card component

fixes FE-1611

### Proposed behaviour
Provide examples of `draggable` with the Card component.
draggable-cards

### Current behaviour
Currently we do not supply examples of draggable Cards in Carbon.

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Additional context
Draggable functionality is only demonstrable in the `Canvas` tab in Storybook.

### Testing instructions
New story created called:
- With draggable

### CODE REVIEW

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by edleeks87 — DISMISSED** [review]

File: src/components/card/card.stories.mdx [hunk 1]
```diff
@@ -528,6 +532,113 @@ Click on the card to see the effect
...
+              <Heading title="Stripe - [account name]" divider={false} />
+              <Typography variant="h5">user.name@sage.com</Typography>
+            </CardColumn>
+            <CardColumn align="right">
+              <Icon type="image" />
+            </CardColumn>
+          </CardRow>
+          <CardRow>
+            <CardColumn>
+              <Typography variant="h5" weight="bold">
```
Comment (jamime) [inline src/components/card/card.stories.mdx:593]: I think we should make these two cards different so you can see they actually changed position.

File: src/components/card/card.stories.mdx [hunk 2]
```diff
@@ -528,6 +532,113 @@ Click on the card to see the effect
   </Story>
 </Preview>
 
+## With draggable
```
Comment (jamime) [inline src/components/card/card.stories.mdx:535]: This example does not work, nothing happens.

Please can you add some more detailed notes, how is this expected to work?

File: src/components/card/card.stories.mdx (hunk 1 above)
Comment (DipperTheDan) [reply src/components/card/card.stories.mdx:593]: Good point. I'll make sure they are different 😃

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by edleeks87 — DISMISSED** [review]

**Review by ZhuoyuJin — DISMISSED** [review]

**Review by divyajindel — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mmaciorowski-sage — APPROVED** [review]

**Review by divyajindel — APPROVED** [review]

**Pull request merged**